sparrowpy/geometry.py

```diff
@@ -279,3 +283,588 @@ def _magnitude(vector):
 
 def _norm(vector):
     return np.array(vector)/_magnitude(np.array(vector))
+
+
+####################################################
+# patch processing (numpy formulation)
+def process_patches(
```

Comment:
```suggestion
def _process_patches(
```
these are all private methods, aren't they?